### PR TITLE
Upgrading xstream version & whitelisting classes for deserialization

### PR DESCRIPTION
**Pull request by abika5**

### CODE REVIEW

File: src/main/java/com/marklogic/contentpump/DocumentMetadata.java [hunk 1]
```diff
@@ -70,6 +70,8 @@ public void setNakedProps(boolean isNakedProps) {
      * @param reader
      */
     public static DocumentMetadata fromXML(Reader reader) {
+        xstream.allowTypes(new Class[] {com.marklogic.contentpump.DocumentMetadata.class});
```
Comment (yunzvanessa) [inline src/main/java/com/marklogic/contentpump/DocumentMetadata.java:73]: The parameter that is passed to allowTypes() is an array of classes. So I think you can combine those two lines.

**Review by yunzvanessa — CHANGES_REQUESTED** [review]
The parameter that is passed to allowTypes() is an array of classes. So I think you can combine those two lines.

**Comment by abika5** [comment]
I will create a new PR.

**Comment by yunzvanessa** [comment]
Hi Abika,

You do not need to create a new PR. Just make new commits to your personal branch and the new commits will be reflected here, and it will keep track of all the commit history.

**Comment by abika5** [comment]
Made the changes suggested

**Comment by yunzvanessa** [comment]
Hi Abika,

If you haven't, please make sure to build mlcp and run unit tests again before I approve it, to prevent typos/errors etc.

Thanks,
Vanessa

**Comment by abika5** [comment]
Sure vanessa, will do now

**Comment by abika5** [comment]
Unit testing passed on local.

**Review by yunzvanessa — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**